src/test/java/io/stargate/sgv2/jsonapi/service/embedding/operation/EmbeddingProviderErrorMessageTest.java

```diff
@@ -175,7 +175,7 @@ public void testIncorrectContentTypeXML() {
               .withSubscriber(UniAssertSubscriber.create())
               .awaitFailure()
               .getFailure();
-      assertThat(exception.getCause())
```

Comment:
This is strange: Quarkus version seems to have changed wrapping of exceptions, affecting 3 test methods in this test class, but nowhere else.